tools/Generate-CodepointWidthsFromUCD.ps1

```diff
@@ -260,21 +248,29 @@ If (-not $NoOverrides) {
     }
 }
 
+$ranges.RemoveAll({ $args[0].Width -eq [CodepointWidth]::Narrow }) | Out-Null
+
+$c = 0
+ForEach($_ in $ranges) {
+    $c += $_.End - $_.Start + 1
+}

```

Comment:
Counting the number of covered codepoints last ensures we pick up any changes due to `RemoveAll`.